src/components/NavigationGroup.svelte

```diff
@@ -1,6 +1,6 @@
 {{#if separator == 'top'}}<hr />{{/if}}
 {{#if group.length}}
-<ul class='coz-nav-group'>
+<ul class='{{`coz-nav-group${wrappingClass}`}}'>
```

Comment:
I agree with @GoOz. I spotted this point but did not mention it. Now that @GoOz has spoken, I can tell that we should better use `coz-nav-group--wrapping` to keep consistency with CozyUI logic.